ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/support/commands.js

```diff
@@ -31,6 +31,7 @@ import "cypress-axe";
 import caseApi from "../api/caseApi";
 import concernsApi from "../api/concernsApi";
 import { AuthenticationInterceptor } from "../auth/authenticationInterceptor";
+import {AuthenticationComponent} from  "../auth/authenticationComponent";
 
```

Comment:
Need to revert these changes, as it will switch it back to the old authentication method